### PR TITLE
rpc: Return incrementalrelayfee in getmempoolinfo

### PR DESCRIPTION
**Pull request by maflcko**
Seems odd to return other policy info, but not the incremental relay fee

### CODE REVIEW

**Review by ghost — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/25439/commits/fafee78188c3de5f6245ec769429822ca4b98c63

**Review by w0xlt — APPROVED** [review]
Code Review ACK https://github.com/bitcoin/bitcoin/pull/25439/commits/fafee78188c3de5f6245ec769429822ca4b98c63

**Review by brunoerg — COMMENTED** [review]
Concept ACK

**Review by jarolrod — COMMENTED** [review]
tACK fafee78188c3de5f6245ec769429822ca4b98c63

```
{
  "loaded": true,
  "size": 1,
  "bytes": 141,
  "usage": 1168,
  "total_fee": 0.00000141,
  "maxmempool": 300000000,
  "mempoolminfee": 0.00001000,
  "minrelaytxfee": 0.00001000,
  "incrementalrelayfee": 0.00001000,
  "unbroadcastcount": 0
}
```

File: src/rpc/mempool.cpp [hunk 1]
```diff
@@ -681,7 +682,8 @@ static RPCHelpMan getmempoolinfo()
                 {RPCResult::Type::NUM, "maxmempool", "Maximum memory usage for the mempool"},
                 {RPCResult::Type::STR_AMOUNT, "mempoolminfee", "Minimum fee rate in " + CURRENCY_UNIT + "/kvB for tx to be accepted. Is the maximum of minrelaytxfee and minimum mempool fee"},
                 {RPCResult::Type::STR_AMOUNT, "minrelaytxfee", "Current minimum relay fee for transactions"},
-                {RPCResult::Type::NUM, "unbroadcastcount", "Current number of transactions that haven't passed initial broadcast yet"}
+                {RPCResult::Type::NUM, "incrementalrelayfee", "minimum fee rate increment for mempool limiting or BIP 125 replacement in " + CURRENCY_UNIT + "/kvB"},
```
Comment (jonatack) [inline src/rpc/mempool.cpp:685]: - s/minimum/Minimum/ (like the other lines in the getmempoolinfo help)
- could add the BTC/kvB unit to minrelaytxfee help

Comment (jonatack) [reply src/rpc/mempool.cpp:685]: (Would be possibly nice for these to be expressed in sats, though I suppose that would require a new RPC.)

**Pull request merged**